docs/configuration.rst

```diff
@@ -8,6 +8,11 @@ Requirements
 * access to Pulp 2 database
 * [recommended] all Pulp 3 supported checksum types are allowed
 
+.. note::
+
+    The migration workload is very SQL-write intensive. It's highly recommended that the volume
```

Comment:
I would just call it IO-intensive.  We know about the worker timeout problem due to SQL writes, but it does a lot of reading and file ops as well, which could influence the system's performance.